Makefile

```diff
@@ -46,11 +46,10 @@ generate_nodejs::
 	$(WORKING_DIR)/bin/$(CODEGEN) nodejs ${VERSION}
 
 build_nodejs:: VERSION := $(shell pulumictl get version --language javascript)
-build_nodejs:: TSC := $(shell which tsc)
 build_nodejs::
 	cd ${PACKDIR}/nodejs/ && \
 		yarn install && \
-		node --max-old-space-size=4096 $(TSC) --diagnostics && \
+		node --max-old-space-size=4096 ./node_modules/.bin/tsc --diagnostics && \
```

Comment:
Also possible to spell `yarn run tsc` if this ever had to run on Windows.

Reply:
I'll look at refactoring this so we can run it consistently - the node args make this a bit trickier to do as either a run command or package.json script if I recall correctly. There is an incantation that'll do it though.